Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx

```diff
@@ -49,12 +49,10 @@ export const ObiEditor: FC<ObiEditorProps> = ({
   );
 
   const deleteLgTemplates = (lgTemplates: string[]) => {
```

Comment:
`deleteLgTemplates` will be defined as `deleteLgText`. See my next comments on `copyLgTemplate`